src/generators/typescript/presets/utils/ExampleFunction.ts

```diff
@@ -0,0 +1,78 @@
+import { TypeScriptRenderer } from '../../TypeScriptRenderer';
+import { CommonModel } from '../../../../models';
+
+/**
+ * LAZY implementation, find first acceptable value
```

Comment:
LAZY? 😄  It's related to draft PR, or you really think about Lazy?

Reply:
Lazy as in find first acceptable example value and use that, you can even change the generation of values.

Reply:
I always think about `Lazy` in programming as of `lazy` initialisation of something, so probably `inferring first acceptable value` will be better to describe the logic of function rather than `Lazy`

Reply:
True, lets be precise 👍